IDEAS/Buildings/Components/BaseClasses/SwWindowResponse.mo

```diff
@@ -25,13 +32,13 @@ model SwWindowResponse "shortwave window respones"
     annotation (Placement(transformation(extent={{-120,0},{-80,40}})));
   Modelica.Blocks.Interfaces.RealInput angInc "angle of incidence"
     annotation (Placement(transformation(extent={{-120,-80},{-80,-40}})));
-  Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_a[nLay] iSolAbs
+  Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_a[nLay] iSolAbs if not removeDynamics
```

Comment:
I would replace this into 
if not removedDynamics and createOutputs
? Also for the other heatports and prescribedheatflows.